ros/api/v1/recommendations.py

```diff
@@ -12,7 +12,10 @@ class RecommendationsApi(Resource):
         'description': fields.String,
         'reason': fields.String,
         'resolution': fields.String,
-        'condition': fields.String
+        'condition': fields.String,
+        'detected_issues': fields.String,
+        'suggested_instances': fields.String,
+        'current_instance': fields.String
```

Comment:
It's good that we are adding new keys as this way it won't break the existing frontend with this change 💯